osu.Game.Rulesets.Mania/ManiaRuleset.cs

```diff
@@ -395,7 +395,7 @@ protected override IEnumerable<HitResult> GetValidHitResults()
 
         public override StatisticItem[] CreateStatisticsForScore(ScoreInfo score, IBeatmap playableBeatmap) => new[]
         {
-            new StatisticItem("Performance Breakdown", () => new PerformanceBreakdownChart(score, playableBeatmap)
+            new StatisticItem("Performance Breakdown", () => new PerformanceBreakdownChart(score, playableBeatmap, showFCPerformance : false)
```

Comment:
Why does mania disable this? You can't just push changes like this unexplained and expect not to get asked about it.

Reply:
mania performance is entirely accuracy based, and not affected by miss count or combo, so showing performance for full combo doesn't really make sense

Reply:
While that is a reasonable explanation it just kinda makes me more opposed to the idea of merging this further to be honest.